frontend/src/app/features/book/components/book-browser/book-filter/book-filter.component.scss

```
@@ -1,14 +1,14 @@
 :host ::ng-deep p-accordion {
```

Stylelint (17.11.1)

[error] Unknown pseudo-element selector "::ng-deep" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)